npbench/benchmarks/polybench/jacobi_2d/jacobi_2d_triton.py

```diff
@@ -46,13 +45,7 @@ def jacobi2d_step(src_ptr, dst_ptr,
 
 def kernel(TSTEPS: int, A: torch.Tensor, B: torch.Tensor):
     assert A.shape == B.shape and A.ndim == 2 and A.shape[0] == A.shape[1]
-    # Force dtype + contiguity to match reference
-    if A.dtype != torch.float64:
-        A = A.to(torch.float64)
-    if B.dtype != torch.float64:
-        B = B.to(torch.float64)
-    A = A.contiguous()
-    B = B.contiguous()
```

Comment:
these checks were unnecessary. an assert makes the code clearer. While the deleted code probably doesn't copy anything, the assert clearly doesn't